docroot/modules/custom/mass_content/includes/mass_content.org_migration.inc

```diff
@@ -130,7 +130,20 @@ function _mass_content_org_page_migration_what_would_you_like_to_do(&$node) {
  * Migrate data for the featured topics section.
  */
 function _mass_content_org_page_migration_featured_topics(&$node) {
-
+  if (!empty($node->field_ref_card_view_6->getValue())) {
+    $paragraph = Paragraph::create([
+      'type' => 'featured_topics',
+      'field_featured_topics_topics' => $node->field_ref_card_view_6->getValue(),
+    ]);
+    $paragraph->save();
+    $pid = $paragraph->id();
+    $pid_revision = $paragraph->getRevisionId();
+    $node->set("field_ref_card_view_6", []);
+    $node->field_organization_sections[] = [
```

Comment:
I created the reusable function _mass_content_org_page_migration_add_section() at the top of this include file to consistently add new sections to the org_page node, if you want to use that here.

Reply:
We also need to create a new organization section paragraph instead of moving the paragraph reference.